Bash_scripts/HW20/roles/hw202/tasks/main.yml

```diff
@@ -0,0 +1,36 @@
+---
+#Get info from files
+- name: "Get variables from file"
+  import_tasks: template.yml
+
+#Copy to group linux
+- name: "Copy files to hosts"
+  become: true
+  become_user: root
+  ansible.builtin.copy:
+   src: "{{ src_curl }}"
+   dest: "{{ dest_curl }}"
+   owner: hw20
+   mode: 0744
+  when: "'linux' in group_names"
+  
+#Install Curl on Ubuntu
+- name: "Install a .deb package "
+  become: true
+  ansible.builtin.apt:
+    deb: "{{ dest_curl }}"
+    state: present
+    update_cache: yes
+    install_recommends: yes
+    ignore_errors: yes
+  when: ansible_facts['os_family'] == "Debian"
+ 
+  #Install Curl on CentOS
+- name: "Install a .rpm package"
+  become: true
+  ansible.builtin.yum:
+    name: "{{ dest_curl }}"
+    state: present
+    update_cache: yes
+    allow_downgrade: yes
+  when: ansible_facts['os_family'] == "RedHat"
```

Comment:
Please do this https://stackoverflow.com/questions/44704968/visual-studio-code-insert-new-line-at-the-end-of-files :) Save your and my time ;)

Reply:
thanks, installed